arch/sim/Kconfig

```diff
@@ -159,6 +159,15 @@ config SIM_HOSTFS
 	---help---
 		Access host filesystem through HostFS.
 
+if SIM_HOSTFS
+config SIM_HOSTFS_PATH_CONVERSION
+bool "Simulated HostFS path conversion"
+default n
+---help---
+	If this option is enabled, the nuttx image folder will be used as the root path
+	when hostfs accesses the file, otherwise $CWD will be used as the root path to search
```

Comment:
my understanding is this option doesn't affect `mount -t hostfs -o fs=/absolutepath /mnt`. right?
this description makes me think otherwise.
especially the term `the root path` gives me an impression it changes the behavior of absolute paths.

Reply:
Regardless of whether this configuration is enabled or not, it will not affect the absolute path.
After executing mount -t hostfs -o fs=/absolutepath /mnt, /mnt is always mapped to /absolutepath

The root path here indicates the starting point to find the file, maybe I should change the description